web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/fpoversikt/FpDtoTjeneste.java

```diff
@@ -163,21 +168,42 @@ private FpSak.BrukerRolle finnBrukerRolle(Fagsak fagsak) {
     }
 
     private Set<FpSak.Søknad> finnFpSøknader(Optional<Behandling> åpenYtelseBehandling, List<MottattDokument> mottatteSøknader) {
-        return mottatteSøknader.stream()
-            .map(md -> {
+        return mottatteSøknader.stream().map(md -> {
                 var status = statusForSøknad(åpenYtelseBehandling, md.getBehandlingId());
                 var perioder = ytelseFordelingTjeneste.hentAggregatHvisEksisterer(md.getBehandlingId()).map(ytelseFordelingAggregat -> {
                     var oppgittFordeling = ytelseFordelingAggregat.getOppgittFordeling();
                     return oppgittFordeling.getPerioder().stream().map(FpDtoTjeneste::tilDto).collect(Collectors.toSet());
                 }).orElse(Set.of());
                 var behandling = behandlingRepository.hentBehandling(md.getBehandlingId());
                 var oppgittDekningsgrad = dekningsgradTjeneste.finnOppgittDekningsgrad(BehandlingReferanse.fra(behandling));
-                return new FpSak.Søknad(status, md.getMottattTidspunkt(), perioder, oppgittDekningsgrad.map(FpDtoTjeneste::tilDekningsgradDto).orElse(null));
-            })
-            .filter(s -> !s.perioder().isEmpty()) //Filtrerer ut søknaden som ikke er registert i YF. Feks behandling står i papir punching
+                var morArbeidUtenDok = morArbeidUtenDok(behandling);
+                return new FpSak.Søknad(status, md.getMottattTidspunkt(), perioder,
+                    oppgittDekningsgrad.map(FpDtoTjeneste::tilDekningsgradDto).orElse(null), morArbeidUtenDok);
+            }).filter(s -> !s.perioder().isEmpty()) //Filtrerer ut søknaden som ikke er registert i YF. Feks behandling står i papir punching
             .collect(Collectors.toSet());
     }
 
+    private boolean morArbeidUtenDok(Behandling behandling) {
+        if (!behandling.getFagsak().getRelasjonsRolleType().erFarEllerMedMor() || mottattEllerVenterDokumentasjonPåAtMorErIArbeid(behandling)) {
+            return false;
+        }
+        var søknadsperioder = ytelseFordelingTjeneste.hentAggregatHvisEksisterer(behandling.getId())
+            .map(yfa -> yfa.getOppgittFordeling().getPerioder())
+            .orElse(List.of());
+        return søknadsperioder.stream().anyMatch(OppgittPeriodeEntitet::erAktivitetskravMedMorArbeid);
+    }
+
+    private boolean mottattEllerVenterDokumentasjonPåAtMorErIArbeid(Behandling behandling) {
+        var vedleggMorsArbeid = søknadRepository.hentSøknadFraGrunnlag(behandling.getId())
+            .map(SøknadEntitet::getSøknadVedlegg)
+            .orElseGet(Set::of)
+            .stream()
+            .filter(v -> Set.of(DokumentTypeId.MOR_ARBEID.getKode(), DokumentTypeId.DOK_MORS_UTDANNING_ARBEID_SYKDOM.getKode())
```

Comment:
her trenger du kun sjekke på MOR_ARBEID - det er det som kommer i søknader siste året.

Reply:
Sorry. Her bør du nok ha den - for å ikke trigge plutselige varsler i revurdering av eldre saker.